scripts/constants.sh

```diff
@@ -2,18 +2,11 @@
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
-# Use lower_case variables in the scripts and UPPER_CASE variables for override
-# Use the constants.sh for env overrides
-
-TELEPORTER_PATH=$(
```

Comment:
What's the reason for removing this and putting it in the other scripts? My thinking was that we could put this here and source constants.sh everywhere that needs it.

Reply:
That was my originally thinking also, but then I realized having `TELEPORTER_PATH` set in each individual script is consistent with the pattern used in the `subnet-evm` scripts.

The reason is that sourcing the `constants.sh` file from a script requires knowing where the script is in relation to to the current working directory the script is being called from. So it's less error prone to set the relative path to the base of the repo from each script individual (no matter where it's called from), and then source `constants.sh` relative from there.

Reply:
Pesky chickens and eggs 😆 Makes sense